src/SFA.DAS.ProviderRegistrations/Services/TrainingProviderApiClient.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using SFA.DAS.ProviderRegistrations.Configuration;
+using SFA.DAS.ProviderRegistrations.Models;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ProviderRegistrations.Services
+{
+    ///<inheritdoc cref="ITrainingProviderApiClient"/>
+    public class TrainingProviderApiClient : ITrainingProviderApiClient
+    {
+        private readonly HttpClient _httpClient;
+        private readonly TrainingProviderApiClientConfiguration _config;
+        private readonly ILogger<TrainingProviderApiClient> _logger;
+
+        public TrainingProviderApiClient(
+            ITrainingProviderApiClientFactory factory,
+            TrainingProviderApiClientConfiguration config,
+            ILogger<TrainingProviderApiClient> logger)
+        {
+            _httpClient = factory.CreateHttpClient();
+            _config = config;
+            _logger = logger;
+        }
+
+        ///<inherit-doc />
+        public async Task<GetProviderSummaryResult> GetProviderDetails(long providerId)
+        {
+            _logger.LogInformation("Getting Training Provider Details for ukprn:{0} returned OK", providerId);
```

Comment:
I'd be inclined to remove this

Reply:
Removed the logger from TrainingProviderApiClient.cs